src/mscorlib/shared/Internal/Runtime/CompilerServices/Unsafe.cs

```diff
@@ -168,6 +168,23 @@ public static bool AreSame<T>(ref T left, ref T right)
             // ret
         }
 
+        /// <summary>
+        /// Returns true if the address (not the value) referenced by <paramref name="left"/> occurs below
+        /// the address referenced by <paramref name="right"/> in the process's virtual address space.
+        /// </summary>
+        [Intrinsic]
+        [NonVersionable]
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static bool IsBelow<T>(ref T left, ref T right)
```

Comment:
Could you please update this to match https://github.com/dotnet/corefx/issues/26451#issuecomment-359887092